src/uUV.cpp

```diff
@@ -1,11 +1,356 @@
-#ifdef USE_MICRO_UV
-
 #include "uUV.h"
 
+#ifndef USE_MICRO_UV
+void uv_close(uv_async_t *handle, uv_close_cb cb) {
```

Comment:
libuv uses dynamic dispatch but uUV uses static dispatch. The redirect is just to avoid having \#ifdef USE_MICRO_UV every time uv_close is called, because libuv requires the uv_handle_t * cast and uUV requires that we do not cast.